bat/tests/09-build-all-delta-packs/description.txt

```diff
@@ -0,0 +1,4 @@
+09-build-all-delta-packs
```

Comment:
You may have to rename this test to "10-build-all-delta-packs" if https://github.com/clearlinux/mixer-tools/pull/492 is merged before this.

Reply:
Yes, I saw a variety of PRs adding tests that would have conflicting version numbers. I think it is easiest to pick the next available number and resolve conflicts when rebasing.